Calibration/HcalCalibAlgos/macros/CalibCorr.C

```diff
@@ -189,13 +189,9 @@ unsigned int truncateId(unsigned int detId, int truncateFlag, bool debug = false
       depth = 1;
     }
   } else if (truncate0 == 9) {
-    //Depths 1 and 2 in HB considered as depth = 1; rest of the depths in HB as depth = 2
-    if (subdet == 1) {
-      if (depth > 1)
-        depth = 2;
-      else
-        depth = 1;
-    }
+    //Depths 1 and 2 considered as depth = 1; rest of the depths are kept as it is
```

Comment:
The comment does not correspond to what is implemented in the code (it should be 
```suggestion
    //Depths 1 and 2 considered as depth = 2; rest of the depths are kept as it is
```
instead)

And in any case the implementation here is quite different from what was before: please check and explain what was updated in HB and HE that requires this change